test/jdk/java/net/Socket/UdpSocket.java

```diff
@@ -133,8 +132,17 @@ public void testMaxSockets() throws Exception {
         }
     }
 
+
     private Socket newUdpSocket() throws IOException {
-        return new Socket(InetAddress.getLoopbackAddress(), 8000, false);
+        Socket s = null;
+
+        try {
+            s = new Socket(InetAddress.getLoopbackAddress(), 8000, false);
+        } catch (BindException unexpected) {
+            System.out.println("BindException caught retry Socket creation");
```

Comment:
Is this appropriate? Writing to stdout? I would remove it, or write to stderr... I am not sure what the standard practice is here.